js/app.js

```diff
@@ -1,6 +1,7 @@
 $(document).ready(function () {
     setHeightOnLoad();
     window.addEventListener('resize', resizeFunction);
+    addition();
```

Comment:
bądź konsekwętny jeśli używasz jquery to korzystaj z jquery zmień ten event listner na jQuerowy z on